src/cfml/system/services/ServerService.cfc

```diff
@@ -1584,11 +1583,17 @@ component accessors="true" singleton {
 			menuItem[ 'path' ] = fileSystemUtil.resolvePath( menuItem.path, relativePath );
 		}
 
+		//checking if the action is the legacy "restartserver", it needs to be translated to a box type action
+		if( menuItem.keyExists( 'action' ) && listFindNoCase('restartserver',menuItem.action)){
+			menuItem[ 'action' ] = "box server restart " & '"#name#"';
+		}
+
 		//need to check if a shell has been defined for this action
 		if( menuItem.keyExists( 'action' ) && listFindNoCase('run,runAsync,runTerminal',menuItem.action)){
 			menuItem[ 'shell' ] = menuItem.shell ?: fileSystemUtil.getNativeShell();
 			// Some special love for box commands
 			if( menuItem.command.lCase().reFindNoCase( '^box(\.exe)? ' )  ) {
+				menuItem.command = reReplaceNoCase(menuItem.command, "^box(\.exe)? ", getSystemSetting( 'java.class.path', 'Unknown' ) & " ", "one")
```

Comment:
Do not allow the binary to be replaced with the text "unknown" as this makes no sense here and would certainly cause an error.  If  `java.class.path` is unavailable, simply do not replace it.